pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopics.java

```diff
@@ -120,8 +120,7 @@ public CmdTopics(PulsarAdmin admin) {
         jcommander.addCommand("set-retention", new SetRetention());
         jcommander.addCommand("remove-retention", new RemoveRetention());
 
-        jcommander.addCommand("enable-deduplication", new EnableDeduplication());
-        jcommander.addCommand("disable-deduplication", new DisableDeduplication());
+        jcommander.addCommand("set-deduplication", new SetDeduplication());
         jcommander.addCommand("get-deduplication-enabled", new GetDeduplicationEnabled());
```

Comment:
shall we use "get-deduplication-enabled" or "get-deduplication"?

Reply:
Well, it sounds good